config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-trusted.yaml

```diff
@@ -47,13 +47,22 @@ postsubmits:
         - cip
         - --thin-manifest-dir=/home/prow/go/src/github.com/kubernetes/k8s.io/k8s.gcr.io
         - --confirm
+        # TODO: these are *not* threads 🤷‍♂️
+        # https://www.geeksforgeeks.org/golang-goroutine-vs-thread/
+        #
+        # this should probably be a multiple of cores below
+        - --threads=14
+        env:
+        - name: GOMAXPROCS
+          value: 7
         resources:
+          # request most of one node 🚀
           requests:
-            cpu: 2
-            memory: "4Gi"
+            cpu: 7
```

Comment:
7, because the node pool has 8 core machines, some is reserved for system daemons etc.